app/src/main/java/com/eveningoutpost/dexdrip/utils/Preferences.java

```diff
@@ -127,9 +130,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
+        preferenceFragment = new AllPrefsFragment();
```

Comment:
Is this change related? it dosnt seem like we use the preferenceFragment again unless Im missing it